app/src/components/common/icons/IconClose.tsx

```diff
@@ -23,17 +23,20 @@ interface Props {
 }
 
 export const IconClose = (props: Props) => {
-  const { color = '#37474F', hoverEffect = false, onClick, size = '24' } = props
+  // If we keep the default value here it prevails on the styled(IconClose) and subsequent styled components I guess. I suggest to remove it
+  const { color, hoverEffect = false, onClick, size = '24', ...restProps } = props
 
   return (
     <Wrapper
-      fill="none"
+      // If we keep 'none' in the fill here then only a hardcoded default variables specified above would apply. Instead, having color here makes the styled(IconClose) color apply (without needing the harcoded value). Here the default value for color always prevails on the styled(IconClose), weird
+      fill={color}
```

Comment:
Can you look maybe more into this seems that we are not passing colors in any of the cases this is probably left from some component we used to have so its maybe fine deleting it from there 
<img width="507" alt="Screenshot 2021-07-05 at 20 07 56" src="https://user-images.githubusercontent.com/33226956/124506794-b1bdb980-ddcc-11eb-9347-17f3a3482b13.png">
I looked into this a bit and I believe we could move the fill property to styled componets like I did so we don't have to hardcode it...What do you think?

Reply:
and please test it out and check that we are not breaking any icons throughout  the codebase